orchestrator-salt/src/test/java/com/sequenceiq/cloudbreak/orchestrator/salt/SaltOrchestratorTest.java

```diff
@@ -744,7 +644,7 @@ public void testNewInstanceIsPrimaryReplacement() throws Exception {
         HostAndRoleTarget replicaTarget = new HostAndRoleTarget("freeipa_replica", Set.of("newInstance.domain", "replica2.domain"));
         ArrayNode replicaRole = mapper.createArrayNode();
         replicaRole.add("freeipa_replica");
-        PowerMockito.when(SaltStates.getGrains(eq(saltConnector), eq(replicaTarget), eq("roles")))
+        lenient().when(saltStates.getGrains(eq(saltConnector), eq(replicaTarget), eq("roles")))
```

Comment:
why `lenient` is required? it's not a parametrized test

Reply:
the `getGrains` called 3 times within the `saltOrchestrator.installFreeIpa`
PowerMockito swallowed this, the new Mockito does not. I can add 3 different `when`s if needed.